src/Inspetor/config.php

```diff
@@ -5,8 +5,8 @@
         'inspetorEnv'                 => false,
         'appId'                       => null,
         'trackerName'                 => null,
-        'collectorHost'               => 'analytics.useinspetor.com',
-        'collectorHostDev'            => 'analytics-dev.useinspetor.com',
+        'collectorHost'               => 'heimdall-prod.inspcdn.net',
```

Comment:
What I am doing is the following:

- collectorHost -> `heimdall-prod.inspcdn.net/prod`
- collectorHostDev -> `heimdall-prod.inspcdn.net/staging`

This way we are always sending to the `prod` version of heimdall, but we can make it send to the to the right snowplow collector

Reply:
cool cool--updated!